core/src/main/scala/kafka/log/LogCleanerManager.scala

```diff
@@ -183,7 +183,7 @@ private[log] class LogCleanerManager(val logDirs: Array[File],
       while (!isCleaningInState(topicPartition, LogCleaningPaused))
         pausedCleaningCond.await(100, TimeUnit.MILLISECONDS)
     }
-    info(s"The cleaning for partition $topicPartition is aborted and paused")
+    info(s"The cleaning for partition $topicPartition is paused")
```

Comment:
As @guozhangwang said, I think the point was to also say that we have aborted. One way to make this clearer is to add a separate log line stating that we are aborting the cleaning for partition above the following line:

```scala
inProgress.put(topicPartition, LogCleaningAborted)
```